### PR TITLE
Gives them syndie boys some new toys #freezeisover

### PR DESCRIPTION
**Pull request by kilkun**
:cl:
experiment: Reports are coming in of Syndicate strike teams deploying experimental super soldiers. Crew are advised to be on alert.
/:cl:

This PR finally puts the corporate species datum into use. Basically, for 90TC (to be tweaked), Ops can buy a modified implanting chamber that will turn one of them into a combat cyborg. Cyborg as in the traditional idea, not the in-game established cyborg race. They also get a swanky space-proof greatcoat, since the whole thing is a reference to Bullfrog's Syndicate agents of the 1993 cult classic, Syndicate.

Also it puts the officer's jacket into use. Previously it was used for a long-deprecated equipment datum. I simply debuffed it's armor values and made it use the greatcoat sprite.

Now that the feature freeze is over, can we finally go over this?

### CODE REVIEW

**Comment by GunHog** [comment]
Syndicate Coursers are a go!

**Comment by kilkun** [comment]
whats ironic is I think that's the catalyst that made me create the species datum originally, as it got me thinking about Bullfrog's Syndicate agents and how well they'd fit in SS13

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -67,19 +67,19 @@ Contains:
...
 	w_class = WEIGHT_CLASS_NORMAL
 	allowed = list(/obj/item/weapon/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/weapon/melee/baton, /obj/item/weapon/restraints/handcuffs, /obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50, energy = 50, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
+	armor = list(melee = 30, bullet = 40, laser = 10, energy = 10, bomb = 0, bio = 100, rad = 100, fire = 100, acid = 100)
 	strip_delay = 130
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 
-	//NASA Voidsuit
+	//NASA Voidsuita
```
Comment (BeeSting12) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Nasa voidsuita?

Comment (kilkun) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: oops

Comment (BeeSting12) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: muh spelling

**Comment by WJohn** [comment]
this sounds like an absolutely gigantic investment of telecrystals for very little gain. why would anyone use this to bank it all on the success of a single operative? we know the l6 saw doesn't work that well because the instant he's down the ops lose like all of their firepower, and it also starves the rest of them of any real weapons.

this is useless. 👎

**Comment by MoreRobustThanYou** [comment]
@WJohn use it on the guy known for being super robust

**Comment by DoctoreMad** [comment]
That made sense on normal SS13 and people usually used a static name because they didn't know they would be a cultist so you'd get "OLDMAN ROBUSTIN WISHES TO BE CULT LEADER" and everyone would press yes.
Nuke Op teams in comparison are composed of 1337 k1ds who will gladly steal their comrades PDA to get 90 points and buy a exosuit.

**Comment by Iamgoofball** [comment]
Sounds like emergent gameplay

**Comment by KorPhaeron** [comment]
My concern with this is there is basically zero way for someone fighting them to know what the hell is going on if they just put on their regular hardsuit and are invisibly immune(?) to stuns and taking almost no damage

**Comment by DoctoreMad** [comment]
There are already some situations where you can't tell why someone is immune to stuns. Traitor Stimpaks probably being the prime one

**Comment by KorPhaeron** [comment]
Those are not good either

**Comment by DoctoreMad** [comment]
Okay look, I don't exactly disagree with you, but I think you should go discuss this with the players instead of blocking this PR because of that.

**Comment by kilkun** [comment]
several points here

1.) There is no stun immunity. If nuke ops want to invest even more TC into their walking engine of destruction, by all means let them. All the adrenals in the world won't make it as effective as a mech which you can buy for even less moolah.

2.) Our armor system is fucking garbage and even stacking the natural laser resistances with the hardsuits, its not terribly hard to lase them to death. 11 shots to send one into crit with a red hardsuit or a softsuit, 15 with an elite hardsuit. This is intentional, you're not going to kill one solo.

3.) Stims don't make you immune to stuns. False advertising.

4.) An ops team can spend 140TC on a augmentation suite, 7 stimpacks and a CNS rebooter on this single operative. Or they can buy a mauler.
A fucking. Mauler.

Plain and simple, a Mauler not only offers real stun immunity, but also all of the advantages that this species has, save for a fuck ton more survivability (29 lasers to kill) and at the cost of being slower.

Biggest difference here is that Mauler also comes with grade A weapons including a fucking rocket launcher.

The only and I mean only reason why you would EVER pick an augmented operative over a Mauler is for the surprise factor. This adds a special element of surprise back into the gamemode. How is that a bad thing?

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -67,19 +67,19 @@ Contains:
...
-	item_state = "det_suit"
+	icon_state = "hos"
+	item_state = "greatcoat"
 	blood_overlay_type = "coat"
 	slowdown = 0
 	flags_inv = 0
 	w_class = WEIGHT_CLASS_NORMAL
 	allowed = list(/obj/item/weapon/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/weapon/melee/baton, /obj/item/weapon/restraints/handcuffs, /obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50, energy = 50, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
+	armor = list(melee = 30, bullet = 40, laser = 10, energy = 10, bomb = 0, bio = 100, rad = 100, fire = 100, acid = 100)
```
Comment (GunHog) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Is this not the Deathsquad officer's coat you are changing here?

Comment (kilkun) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Deathsquad officers get hardsuits not coats

It's a very old equipment datum that hasn't been used in, like, half a decade

Comment (ShizCalev) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: It's used by Special Ops Officers, and the coat is present in lockers on CentComm.

**Comment by banthebantz** [comment]
Agents are arguably the most OP race with their relentless superspeed and esword punches that stun but for 50TC you can just get a syndicate borg which is much, much better.

**Comment by InsaneHyena** [comment]
Just like your previous PRs (#15567), this is absolute trash, and I suggest you to stop coding.

**Comment by kevinz000** [comment]
Nice so cqc but better
:-1:

**Comment by kilkun** [comment]
cqc is lame

**Comment by kilkun** [comment]
Too many negative comments. More features to be added before it comes back, IF it comes back.

**Comment by KorPhaeron** [comment]
I think its a cool thing to reference and a cool thing lorewise for the record its just gonna be hard to make it interesting/well balanced

**Comment by kilkun** [comment]
It most likely will come back at some point.

I just have other things in the works for tgstation that I want to finish first before I get started on fleshing this out more. Trying to keep the number of complex projects down to prevent burnout.

**Pull request closed without merging**